dali/benchmark/decoder_bench.cc

```diff
@@ -308,5 +311,89 @@ BENCHMARK_REGISTER_F(DecoderBench, HostDecoderSlice)->Iterations(100)
 ->UseRealTime()
 ->Apply(PipeArgs);
 
+BENCHMARK_DEFINE_F(DecoderBench, nvJPEGDecoderRandomCrop)(benchmark::State& st) { // NOLINT
+  int batch_size = st.range(0);
+  int num_thread = st.range(1);
+  DALIImageType img_type = DALI_RGB;
+
+  this->DecoderPipelineTest(
+    st, batch_size, num_thread, "gpu",
+    OpSpec("nvJPEGDecoderRandomCrop")
+      .AddArg("device", "mixed")
+      .AddArg("output_type", img_type)
+      .AddInput("raw_jpegs", "cpu")
+      .AddOutput("images", "gpu"));
+}
+
+BENCHMARK_REGISTER_F(DecoderBench, nvJPEGDecoderRandomCrop)->Iterations(100)
+->Unit(benchmark::kMillisecond)
+->UseRealTime()
+->Apply(PipeArgs);
+
+BENCHMARK_DEFINE_F(DecoderBench, nvJPEGDecoderCrop)(benchmark::State& st) { // NOLINT
```

Comment:
I think these `// NOLINT`s are the remnant of the dark times when we bowed to cpplint and it's grudge held against non-const l-value references.

Reply:
Removed (all from this file)